src/ucp/api/ucp.h

```diff
@@ -853,7 +853,11 @@ ucs_status_t ucp_worker_wait(ucp_worker_h worker);
  *
  * @param [in]  worker    Worker of notified events.
  *
- * @return Error code as defined by @ref ucs_status_t
+ * @return ::UCS_OK        The operation completed successfully. File descriptor
+ *                         will be signaled by new events.
+ * @return ::UCS_ERR_BUSY  Some events arrived before the operation and has
+ *                         been cleaned-up from notification queue.
```

Comment:
What is the user workflow in this case ? How we expect users to deal with this ? I think we should cover this in documentation